src/main/java/org/kizombadev/markdownparser/SyntaxAnalyzer.java

```diff
@@ -46,32 +75,55 @@ public Syntax parse(ImmutableList<Token> tokens) {
         return root;
     }
 
-    private void handleQuotation(RootSyntax currentRoot) {
-        Syntax quotation = new Quotation();
+    private void handleUnorderedList(MutableSyntax currentRoot) {
+        MutableSyntax unorderedList = new UnorderedListSyntax();
+
+        while (currentToken() == Token.Star && nextToken() == Token.Blank) {
+            stepTokenForward();
+            stepTokenForward();
+
+            handleUnorderedListItem(unorderedList);
+
+            if (currentToken() == Token.NewLine) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Use the "equals" method if value comparison was intended. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1698)